Dockerfile

```diff
@@ -94,21 +91,35 @@ ADD requirements.txt /requirements.txt
 ADD dev-requirements.txt /dev-requirements.txt
 # Install dependencies
 # TODO: Add --std=c++14
+ARG GPU_ARCH=";"
+ARG PREFIX=/usr/local
+ARG USE_FIN="OFF"
+ARG CCACHE_SECONDARY_STORAGE="redis://10.236.118.172"
+# RUN cget -p $PREFIX install https://github.com/ccache/ccache/archive/7f1572ae9ca958fa923a66235f6a64a360b03523.tar.gz -DZSTD_FROM_INTERNET=ON -DHIREDIS_FROM_INTERNET=ON
+RUN rm -rf /tmp/ccache* && mkdir /tmp/ccache && wget https://github.com/ccache/ccache/archive/7f1572ae9ca958fa923a66235f6a64a360b03523.tar.gz -O /tmp/ccache.tar.gz && \
```

Comment:
I assume you are not using cget here because rbuild cleans out the packages before building, but you can list this in the rbuild.ini file so it will install for the develop and fin sessions.

Reply:
The issue is that I want to use ccache while building the stuff in rbuild.ini that is why I install it directly